README.md

```diff
@@ -578,11 +578,16 @@ export const rootReducer = combineReducers({
 
 **Sagas**
 
-Add the `authorizationSaga` to the `rootSaga`:
+Use `createAuthorizationSaga` and add the returned saga to the `rootSaga`:
 
 ```ts
 import { all } from 'redux-saga/effects'
-import { authorizationSaga } from 'decentraland-dapps/dist/modules/authorization/sagas'
+import { createAuthorizationSaga } from 'decentraland-dapps/dist/modules/authorization/sagas'
+
+// if you are not going to use meta transactions you can skip the options or just import `authorizationSaga`
```

Comment:
Maybe we can put this above as part of the explanation? Maybe it's easy to miss here (not that anyone reads READMEs anyways :P)

Reply:
done ✅